utils/include/utils.hpp

```diff
@@ -9,6 +9,11 @@ namespace util {
 
 bool parseTTL(const std::string& ttlString, uint64_t& ttl);
 
+bool parseTimestamp(const std::string& timestampString, const uint64_t& ttl,
+                    uint64_t& timestamp);
+
+uint64_t get_time_ms();
```

Comment:
We should aim to comment our code, especially functions in utils which are meant to be reused.

Reply:
:+1: